### DISCUSSION TITLE
Lesson4:Fund Me(doubt)

### DISCUSSION THREAD

**Discussion by yasharyan0904** [Cyfrin/foundry-full-course-cu#1753] [Q&A] [answered]
Why did we take minUSD as 5e18 initially because the getConversionRate is already giving the value in uint256 as the real value of eth in USD (as we are dividing by 1e18 already in the code)??
Why can't we take minUSD as 5 simply??

**Comment by urosognjenovic** [comment, marked as answer]
Let's take some values as an example.

If you call the function getConversionRate passing 0.01 for ```ethAmount```, ```ethPrice``` will be 3810e18 (current price) and ```ethAmountInUsd``` will be ```(ethPrice * ethAmount) / 1000000000000000000``` = ```3810e18 * 0.01e18 / 1e18``` = ```38.1e18``` and the minimum USD amount is 5e18.

**Reply by yasharyan0904** [reply]
Thanks a lot, I didn't considered eth amount in wei .
As, the amount in msg.value is always in wei.
